### PR TITLE
Internalize WPAVE and WWPAVE keywords

### PR DESCRIPTION
**Pull request by joakim-hove**
Load and internalize WPAVE and WWPAVE keywords. These keywords are loaded as a first step towards supporting the `WBP?` keywords - to complete the `WBP?` work is a *large task*.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Opinions on this?

**Review by bska — COMMENTED** [review]
It's a good start and I think it should go in.  I just had a couple of questions concerning the use of `make_shared<>()` and the serialization tests.

File: src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp [hunk 1]
```diff
@@ -1755,6 +1755,27 @@ namespace {
         }
     }
 
+    void Schedule::handleWPAVE(const HandlerContext& handlerContext, const ParseContext&, ErrorGuard&) {
+        auto wpave = std::make_shared<PAvg>( PAvg( handlerContext.keyword.getRecord(0) ));
```
[the remark this answers is not in the document]
Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp:1759]: No - I'll update

File: opm/parser/eclipse/EclipseState/Schedule/Well/PAvg.hpp [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    template<class Serializer>
+    void serializeOp(Serializer& serializer) {
+        serializer(m_inner_weight);
+        serializer(m_conn_weight);
+        serializer(m_depth_correction);
+        serializer(m_open_connections);
+    }
+
+    static PAvg serializeObject() { return PAvg(); }
```
[the remark this answers is not in the document]
Comment (joakim-hove) [reply opm/parser/eclipse/EclipseState/Schedule/Well/PAvg.hpp:55]: OK - I'll make up something

**Review by bska — APPROVED** [review]
This looks good to me.  I'll run a build check and merge when successful.

**Comment by bska** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**